src/app/voting/voting.tpl.html

```diff
@@ -15,6 +15,7 @@
                     <td>{{ vote.song.artist || "-" }}</td>
                     <td >
                         <div class="pull-right">
+                            <i ng-if="isSongInQueue(vote.song)" class="fa fa-clock-o fa-lg"></i>
```

Comment:
you can set the title attribute like: `title="Will be played soon."` or "Is in DJ queue.". This text will be displayed, if someone hovers the icon. Maybe this works on mobile devices too (I haven't tested it yet).